src/Plutarch/Extra/AssetClass.hs

```diff
@@ -150,9 +158,9 @@ passetClass = phoistAcyclic $
   plam $ \sym tk ->
     pcon $ PAssetClass (pdata sym) (pdata tk)
 
--- | @since 3.10.0
+-- | @since 3.10.3
 passetClassT ::
-  forall (unit :: Symbol) (s :: S).
```

Comment:
these can be given `(unit :: k)` maybe? or `{k} (unit :: k)` if the first one doesn't work.